content/docs/command-reference/plots/index.md

```diff
@@ -158,6 +158,63 @@ header (first row) are equivalent to field names.
 
 - `<DVC_METRIC_Y_TITLE>` (optional) - field name to display as the X axis label
 
+## HTML templates
+
+By default, the plots generated by `dvc plots` require Internet access to
+provide Vega JavaScript libraries. It's possible to supply a custom HTML file to
+`dvc plot show` with `--html-template` option when Internet access is not
+available
+
+> ⚠️ This is a separate feature from
+> [custom plot templates](/doc/command-reference/plots#custom-templates) to
+> provide a custom HTML file for the _generated plots._
+
+You can create an HTML file with `<script>` references to Vega libraries and put
+`{plot_divs}` as a placeholder for the generated plots. You can supply this HTML
+file with `--html-template` option.
+
+### Example for the custom HTML template
+
+First, download the Vega libraries and put them to a directory you produce the
```

Comment:
This part is too long IMO. And basically the same as the new guide being introduced. Let's not have all this in the cmd desc? Esp. since it's repeated. Can we summarize more? The desc. for now should just explain I think. Eventually (#2572) we'll extract to the UG properly.

Reply:
The guide won't be there. I'll delete the last sentence as it repeats the option.

Reply:
> I'll delete the UG file if everything looks all right.

I see the intention is to only have the example here. OK how about an actual **Example** instead of making the ref description so long?

Reply:
p.s. the example text may need a bit of copy editing similar to what I mentioned in https://github.com/iterative/dvc.org/pull/2472#discussion_r656806369. BTW I can help with that if needed.

Reply:
moving example to a section below?

I didn't do to keep it separate from other examples but I can move to the bottom.

Reply:
Yep, moving it after https://dvc.org/doc/command-reference/plots#example-confusion-matrix basically. To the very bottom of the doc. basically (but easy to find with the right ToV nav) + link from the desc.

Reply:
I did this in b86707b  and 7e966f9.